definitions/variable/emissions/emissions.yaml

```
@@ -248,21 +284,24 @@
     unit: "{Level-3 Species}"
 
 - Emissions|{Level-2 Species}|Waste:
-    description: Emissions of {Level-2 Species} from waste (IPCC category 6)
-    unit: "{Level-2 Species}"
-
-- Emissions|{Level-2 Species}|Fossil Fuel Fires:
-    description: Emissions of {Level-2 Species} from fossil fuel fires (IPCC category 7)
+    description: Emissions of {Level-2 Species} from waste (IPCC 1996 category 6; IPCC 2006 category 4). 
+      Any emissions from fossil-based products (incineration or CO2 decomposition) should be accounted for here.
+      CO2 from organic waste handling and decay should not be included.
     unit: "{Level-2 Species}"
 
 - Emissions|{Level-2 Species}|Other:
-    description: Emissions of {Level-2 Species} from other sources
+    description: Emissions of {Level-2 Species} from other sources, (IPCC 1996 category 7; IPCC 2006 category 5B; including fossil fuel fires). 
+      Only use this category exceptionally, for any categories than cannot be accommodated in the categories described above. 
+      Unless specified otherwise, this will be treated as a flux from some fossil reservoir to the atmosphere in climate models.
     unit: "{Level-2 Species}"
 
-- Emissions|{Level-3 Species}|Capture and Removal:
-    description: Capture and removal of atmospheric {Level-3 Species} using net-negative
-      technologies that are not directly linked to an emissions source,
-      e.g., direct air capture (DAC) or enhanced weathering
+- Emissions|{Level-3 Species}|Other Capture and Removal:
+    description: Capture and removal of atmospheric {Level-3 Species} using other net-negative 
+      technologies that are not reported under other variables as they are not directly linked 
```

Removing this extra qualifier improves readibility.
```suggestion
      technologies that are not directly linked 
```

---

Sure, I thought for me it would be helpful (this is how I would explain it to others) but fine.